src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,215 @@
+    public Car update(Car car) {
+        String updateCarRequest = "UPDATE taxi_service_db.cars SET "
+                + "cars.model = ?, cars.manufactures_id = ? "
+                + "WHERE cars.id = ? AND cars.is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement updateCarStatement
+                        = connection.prepareStatement(updateCarRequest)) {
+            updateCarStatement.setString(1, car.getModel());
+            updateCarStatement.setLong(2, car.getManufacturer().getId());
+            updateCarStatement.setLong(3, car.getId());
+            updateCarStatement.executeUpdate();
+            return car;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't update "
+                    + car + " in cars DB.", e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteCarRequest = "UPDATE cars SET is_deleted = TRUE WHERE id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement deleteCarStatement
+                        = connection.prepareStatement(deleteCarRequest)) {
+            deleteCarStatement.setLong(1, id);
+            return deleteCarStatement.executeUpdate() != 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't delete car with id " + id, e);
+        }
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        String getAllByDriverRequest = "SELECT cars.id, cars.model, cars.manufactures_id, "
+                + "drivers.id, "
+                + "drivers.name, drivers.licenseNumber, manufacturers.name, "
+                + "manufacturers.country FROM taxi_service_db.cars_drivers "
+                + "JOIN taxi_service_db.cars "
+                + "ON cars.id = cars_drivers.car_id "
+                + "JOIN taxi_service_db.drivers "
+                + "ON drivers.id = cars_drivers.driver_id "
+                + "JOIN taxi_service_db.manufacturers "
+                + "ON cars.manufactures_id = manufacturers.id "
+                + "WHERE cars_drivers.driver_id = ? "
+                + "AND drivers.is_deleted = FALSE;";
+        List<Car> cars = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getAllByDriverStatement
+                        = connection.prepareStatement(getAllByDriverRequest)) {
+            getAllByDriverStatement.setLong(1, driverId);
+            ResultSet resultSet = getAllByDriverStatement.executeQuery();
+            while (resultSet.next()) {
+                cars.add(parseCarsFromResultSet(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get a list of cars from driversDB.", e);
+        }
+        for (Car car : cars) {
+            getDriversForCar(car.getId());
```

Comment:
you get the list of drivers for car but don't set it to your car.
You sould set list of drivers for every car in for loop